### DISCUSSION TITLE
Does Sumatra have a Dark Mode setting?

### DISCUSSION THREAD

**Discussion by martinkirlow** [sumatrapdfreader/sumatrapdf#3413] [General]
Does it have Dark Mode or Light Mode setting?

**Comment by GitHubRulesOK** [comment]
No why would it? its only an image canvas with minimal surround. press I key to invert
![image](https://user-images.githubusercontent.com/12754913/229310441-3a2d10a3-a67a-4bd1-b7b0-74abc538a127.png)

However watch this space, closely, as many interface colours have been recently adjusted, So much is Work In Progress.

**Comment by martinkirlow** [comment]
I prefer the dark(ish) as I read books, mostly. The 'I' key is what I need. :)
